azure-pipelines-pr.yml

```diff
@@ -82,6 +82,29 @@ stages:
         - task: ComponentGovernanceComponentDetection@0
           displayName: Component Governance scan
 
+      - job: MacOS
```

Comment:
Do we actually want signing for the PR builds?

Reply:
Signing in PRs is always in dry run mode. Having this enabled helps us to validate this part of the infra:

```
SignToolTask starting.
DryRun: True
Signing mode: Real
MicroBuild signing logs will be in (Signing*.binlog): /Users/runner/work/1/s/artifacts/log/Debug/
MicroBuild signing configuration will be in (Round*.proj): /Users/runner/work/1/s/artifacts/tmp/Debug/
Round 0: Signing 1 files.
Repacking 1 containers.
Repacking 1 containers in parallel.
Round 1: Signing 1 files.
Build artifacts signed and validated.
SignToolTask execution finished.
Publishing symbol packages to MSDL ...
Performing symbol publish  (Dry Run)..
SymbolServerPath: https://microsoftpublicsymbols.artifacts.visualstudio.com/DefaultCollection
ExpirationInDays specified: 3650
Symbol publish finished
Publishing symbol packages to SymWeb ...
Performing symbol publish  (Dry Run)..
SymbolServerPath: https://microsoft.artifacts.visualstudio.com/DefaultCollection
ExpirationInDays specified: 3650
Symbol publish finished
```